apiserver/facades/agent/provisioner/provisioner.go

```diff
@@ -1561,7 +1449,7 @@ func (p *ProvisionerAPI) SetModificationStatus(args params.SetStatus) (params.Er
 }
 
 func (p *ProvisionerAPI) setOneModificationStatus(canAccess common.AuthFunc, arg params.EntityStatusArgs) error {
-	logger.Tracef("SetInstanceStatus called with: %#v", arg)
+	logger.Tracef("SetModificationStatus called with: %#v", arg)
```

Comment:
So the ModificationStatus() method above raises "NotSupported" and yet we've retained this method it used to call but which appears no longer used now?

Reply:
setOneModificationStatus() is called by SetModificationStatus() which is being kept, not ModificationStatus().